### PR TITLE
Disallow generating audit report before audit starts

### PR DESCRIPTION
**Pull request by jonahkagan**

### CODE REVIEW

File: server/api/reports.py [hunk 1]
```diff
@@ -742,6 +743,9 @@ def sampled_batch_rows(election: Election, jurisdiction: Jurisdiction = None):
 @api.route("/election/<election_id>/report", methods=["GET"])
 @restrict_access([UserType.AUDIT_ADMIN])
 def audit_admin_audit_report(election: Election):
+    if len(list(election.rounds)) == 0:
```
Comment (carolinemodic) [inline server/api/reports.py:746]: is election.rounds the current round of an audit we are on or something?

File: server/tests/api/test_reports.py [hunk 2]
```diff
@@ -77,3 +77,35 @@ def test_jurisdiction_admin_report(
         == 'attachment; filename="audit-report-J1-Test-Audit-test-jurisdiction-admin-report-DATETIME.csv"'
     )
     assert_match_report(rv.data, snapshot)
+
+
+def test_report_before_audit_starts(
```
Comment (carolinemodic) [inline server/tests/api/test_reports.py:82]: just curious do we have something like jest coverage that tracks the test coverage in ARLO?

**Review by carolinemodic — APPROVED** [review]

File: server/api/reports.py (hunk 1 above)
Comment (jonahkagan) [reply server/api/reports.py:746]: `election.rounds` is all of the rounds of the audit. so if it's empty, that means the audit hasn't started yet

File: server/tests/api/test_reports.py (hunk 2 above)
Comment (jonahkagan) [reply server/tests/api/test_reports.py:82]: yeah, it's configured in .coveragerc. currently at 99.6%. i'd like to get to 100% - would probably only take a few hours

**Pull request merged**